docs/machine-learning/resources/glossary.md

```diff
@@ -15,71 +15,71 @@ The following list is a compilation of important machine learning terms that are
 
 ## Accuracy
 
-The proportion of true results to total cases. Ranges from 0 (least accurate) to 1 (most accurate). Accuracy is only one evaluation measure used to score performance of your model and should be considered in conjunction with [precision](#precision) and [recall](#recall).
+In [classification](#classification), accuracy is the number of correctly classified items divided by the total number of items in the test set. Ranges from 0 (least accurate) to 1 (most accurate). Accuracy is one of evaluation metrics of the performance of your model. Consider it in conjunction with [precision](#precision), [recall](#recall), and [F-score](#f-score).
 
```

Comment:
This is fine for now, but we'll need to break this down into accuracy for binary classification and micro and macro accuracy for multi-classification later.